core/src/main/scala/com/softwaremill/kmq/redelivery/streams/RedeliveryTracker.scala

```diff
@@ -0,0 +1,35 @@
+package com.softwaremill.kmq.redelivery.streams
+
+import akka.actor.ActorSystem
+import akka.kafka.ConsumerSettings
+import com.softwaremill.kmq.{KafkaClients, KmqConfig, MarkerKey, MarkerValue, ParititionFromMarkerKey}
+import com.typesafe.scalalogging.StrictLogging
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.common.serialization.Deserializer
+
+import java.io.Closeable
+import scala.concurrent.{Await, ExecutionContext}
+import scala.concurrent.duration.DurationInt
+
+object RedeliveryTracker extends StrictLogging {
+  def start(bootstrapServers: String, config: KmqConfig): Closeable = {
+    implicit val system: ActorSystem = ActorSystem("kmq-redelivery")
+    implicit val ec: ExecutionContext = system.dispatcher
+    implicit val markerKeyDeserializer: Deserializer[MarkerKey] = new MarkerKey.MarkerKeyDeserializer()
+    implicit val markerValueDeserializer: Deserializer[MarkerValue] = new MarkerValue.MarkerValueDeserializer()
+
+    val markerConsumerSettings = ConsumerSettings(system, markerKeyDeserializer, markerValueDeserializer)
+      .withBootstrapServers(bootstrapServers)
+      .withGroupId(config.getRedeliveryConsumerGroupId)
+      .withProperty(ProducerConfig.PARTITIONER_CLASS_CONFIG, classOf[ParititionFromMarkerKey].getName)
+
+    val streamControl = new RedeliveryAndCommitMarkerStream(markerConsumerSettings,
+      config.getMarkerTopic, 64,
```

Comment:
why 64? it's not unusual to have many more partitions

Reply:
I think we should just use `Int.MaxValue` there so that we create as many streams as there are partitions. The number of sub-streams is bounded anyway.

Reply:
Okey, I'm changing to Int.MaxValue.